tests/Unit/Infrastructure/Processing/AssetProcessingJobWorkerTest.php

```
@@ -30,7 +30,7 @@ final class AssetProcessingJobWorkerTest extends TestCase
 {
     private AssetRepositoryInterface&MockObject $assets;
     private AssetProcessorInterface&MockObject $assetProcessor;
-    private AssetTerminalStatusCacheInterface&MockObject $assetTerminalStatusCache;
+    private AssetStatusCacheInterface&MockObject $assetTerminalStatusCache;
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $assetTerminalStatusCache. Keep variable name length under 20. (undefined)

(LongVariable)